oauth2_http/java/com/google/auth/oauth2/ComputeEngineCredentials.java

```diff
@@ -430,6 +436,12 @@ public IdToken idTokenWithAudience(String targetAudience, List<IdTokenProvider.O
       throw new IOException(METADATA_RESPONSE_EMPTY_CONTENT_ERROR_MESSAGE);
     }
     String rawToken = response.parseAsString();
+
+    LoggingUtils.log(
+        LOGGER_PROVIDER,
+        Level.FINE,
+        ImmutableMap.of("id-token", rawToken),
```

Comment:
Is there a naming convention for the key here? hyphens to seperate the words, camelCase?

Reply:
Good point. I don't think there is not a generic naming convention. However, I agree we should keep consistent as possible. Following most camelCases in [this doc](https://docs.google.com/document/d/1nh1m0o8RADN7wx1w4GFtIa5vpLOIHlNOy9e0ufXItks/edit?resourcekey=0-0dGUiO68s6aMOAHvrMYhYg&tab=t.0#heading=h.swusf33l2yl2), I am going to change this to "idToken"